.circleci/config.yml

```diff
@@ -208,16 +204,35 @@ workflows:
     jobs:
       - prep_env
 
+      # New step: Build musl-cross for x86
```

Comment:
IMO, remove both "New step" comments.  It's not new once it's committed :slightly_smiling_face:

edit: And the "Changed dependency to use the new x86-musl-cross step" comments below

Reply:
@JonathonHall-Purism fixed under [94ee5ec](https://github.com/linuxboot/heads/pull/1734/commits/94ee5ec576a5bdf7f6665b018d8658c7e19d4683) thank

Reply:
@JonathonHall-Purism fixed under [43f3570](https://github.com/linuxboot/heads/pull/1734/commits/43f35702883e73bb12d9d584833239400df861b7)